tools/license-builder.sh

```diff
@@ -100,4 +100,7 @@ addlicense "brotli" "deps/brotli" "$(cat ${rootdir}/deps/brotli/LICENSE)"
 
 addlicense "HdrHistogram" "deps/histogram" "$(cat ${rootdir}/deps/histogram/LICENSE.txt)"
 
+addlicense "node-heapdump" "src/heap_utils.cc" \
+           "$(curl -sL https://raw.githubusercontent.com/bnoordhuis/node-heapdump/0ca52441e46241ffbea56a389e2856ec01c48c97/LICENSE)"
```

Comment:
Can you run `tools/license-builder.sh` + commit the result before merging this?

Reply:
Yes, once I'm back on my ubuntu dev machine, I definitely will :-)